wpmautic.php

```diff
@@ -111,7 +111,7 @@ function wpmautic_injector() {
 	$script_location = wpmautic_option( 'script_location' );
 	if ( 'header' === $script_location ) {
 		add_action( 'wp_head', 'wpmautic_inject_script' );
-	} elseif ( 'footer' === $script_location ) {
+	} elseif ( 'footer' === $script_location || 'disabled' === $script_location ) {
```

Comment:
Maybe just using a `else` here is simpler ?

Reply:
done